galleries/users_explain/quick_start.py

```diff
@@ -111,6 +111,26 @@
 # Artists are drawn to the **canvas**.  Most Artists are tied to an Axes; such
 # an Artist cannot be shared by multiple Axes, or moved from one to another.
 #
+# .. _plot_types_quickstart:
+#
+# Plot types
+# ==========
+#
+# For an overview of the different types of plot you can create with Matplotlib,
+# see the :ref:`Plot types gallery <plot_types>`. If you don't find the plot
+# type you need, it may still be possible to create the visualization you want
+# by combining existing Artists and plot elements.
+#
+# * To manipulate and customize points, see :class:`~.markers.MarkerStyle`
```

Comment:
I think this is a reasonable place for this.  But not a fan of this almost exhaustive list of Artists, under plot types.  Maybe just link the Artists page and have a few examples? eg

```
Matplotlib plotting methods return a variety of :ref:`Artists <users_artists>` that allow customization of the plot elements.  For example, `~.matplotlib.axes.Axes.plot` returns a :class:`~.lines.Line2D` object and `~.matplotlib.axes.Axes.scatter` returns a :class:`~.collections.LineCollection`.  
```

and maybe include a short snippet example.